src/main/webapp/app/pages/usageAnalysisPage/UserUsageDetailsPage.tsx

```diff
@@ -43,24 +44,42 @@ export default class UserUsageDetailsPage extends React.Component<{
       const result = new Map<string, UsageRecord[]>();
```

Comment:
Could you do me a favor to fix the line `<Link to={`${PAGE_ROUTE.USER}/${this.user.userEmail}`}>`? I can't leave a comment there as there is no change comparing to master. But the link is wrong, it populates as /users/:login/useremail, but it should be /users/useremail